File: src/main/java/me/yellowbear/uwujobs/jobs/BlockBreak.java
package me.yellowbear.uwujobs.jobs;

public enum BlockBreak {
    MINER,
    LUMBER,
    FARMER,
    SHOVELER;
    public static BlockBreak getJob(String job) {
        for (BlockBreak jobs : BlockBreak.values()) {
            if (jobs.name().equalsIgnoreCase(job)) {
                return jobs;
            }
        }
        return null;
    }
}

File: src/main/java/me/yellowbear/uwujobs/jobs/BlockBreak.kt
package me.yellowbear.uwujobs.jobs

enum class BlockBreak {
    MINER,
    LUMBER,
    FARMER,
    SHOVELER;

    companion object {
        fun getJob(job: String?): BlockBreak? {
            for (jobs in entries) {
                if (jobs.name.equals(job, ignoreCase = true)) {
                    return jobs
                }
            }
            return null
        }
    }
}

